src/html/components/modal-box.html

```diff
@@ -0,0 +1,20 @@
+<div class="modal-overlay" id="modal-main">
+	<div class="modal-content">
+		<button class="modal-close-button">
+			<span class="modal-close-button__line"></span>
+			<span class="modal-close-button__line"></span>
+		</button>
+		{% include "html/components/registration-block.html" %}
+	</div>
+	<div class="gratitude">
```

Comment:
make isolated block for `<div class="gratitude">` and include it here and to `modal-gratitude.html`

Reply:
Done